src/go/k8s/tests/e2e/node-select-tolerations/01-label-node.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+- command: kubectl label nodes kind-worker2 redpanda-node=true
```

Comment:
nit: Should we also remove the label at the end? Since it needs to be required by pods I doubt it would make any difference but it would be nice to leave a clean state (since we can do it).